engines/router/missionctl/testdata/bq_turing_result_schema.json

```diff
@@ -1,87 +1,120 @@
 [
```

Comment:
Generated from the proto using https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema